src/pudl/transform/eia861.py

```diff
@@ -727,10 +731,10 @@ def _tidy_class_dfs(
     # of tables.
     data_dupe_mask = data_cols.duplicated(subset=idx_cols + [class_type], keep=False)
     data_dupes = data_cols[data_dupe_mask]
-    fraction_data_dupes = len(data_dupes) / len(data_cols)
+    fraction_data_dupes = len(data_dupes) / len(data_cols) if len(data_cols) else 0
     denorm_dupe_mask = denorm_cols.duplicated(subset=idx_cols, keep=False)
     denorm_dupes = denorm_cols[denorm_dupe_mask]
-    fraction_denorm_dupes = len(denorm_dupes) / len(data_cols)
+    fraction_denorm_dupes = len(denorm_dupes) / len(data_cols) if len(data_cols) else 0
```

Comment:
probably want to add the same change to line 739

Reply:
I think it's okay there because it's just printing out the strings of the two numbers, not actually doing the division. And it could be nice to see the actual numbers even in this case.